command/src/logging/logs.rs

```diff
@@ -245,7 +260,7 @@ impl InnerLogger {
                     match backend {
                         LoggerBackend::Stdout(stdout) => {
                             let _ = stdout.write(bytes);
-                            return;
+                            return true;
```

Comment:
For other log backends (Tcp, Unix, Udp…) we keep track of `io_result` in order to log it and revive the said backend.
When writing to Stdout, however,there is no reviving to do. Returning early makes sense.

The `log_access` function returns a boolean because it is called by logging macros. More on that in another discussion in this review.